runtime/autoload/provider/pythonx.vim

```diff
@@ -30,20 +41,16 @@ function! provider#pythonx#Require(host) abort
     call add(args, plugin.path)
   endfor
 
-  try
-    let channel_id = jobstart(args, s:job_opts)
-    if rpcrequest(channel_id, 'poll') ==# 'ok'
-      return channel_id
-    endif
-  catch
-    echomsg v:throwpoint
-    echomsg v:exception
-    for row in get(s:stderr, channel_id, [])
-      echomsg row
-    endfor
-  endtry
-  throw remote#host#LoadErrorForHost(a:host.orig_name,
-        \ '$NVIM_PYTHON_LOG_FILE')
+  let opts = deepcopy(s:job_opts)
+  let opts.orig_name = a:host.orig_name
+  let channel_id = jobstart(args, opts)
+  if channel_id == 0
+    echoerr 'job table is full' 
```

Comment:
scripts should never need to check this. If it's not already, we should make it an error rather than silently returning 0.